code/WorkInProgress/Cael_Aislinn/sculpture.dm

```diff
@@ -291,21 +292,3 @@
 
 //You cannot destroy SCP-173, fool!
 /mob/living/simple_animal/sculpture/ex_act(var/severity)
-
-// Commented this out because it LITERALLY PREVENTS ADMINS FROM REMOVING IT ENTIRELY
-//But seriously, you just can't
-// /mob/living/simple_animal/sculpture/Del()
-//
-// 	replace_SCP()
```

Comment:
It was https://github.com/d3athrow/vgstation13/blame/5504baa68a512a087520a7502ebc67cd78f692df/code/WorkInProgress/Cael_Aislinn/sculpture.dm#L290